.changeset/violet-meals-work.md

```
@@ -0,0 +1,45 @@
+---
+"@seed-design/tailwind3-plugin": minor
+"@seed-design/migration-index": minor
+"@seed-design/tailwind4-theme": minor
+"@seed-design/rootage-artifacts": minor
+"@seed-design/css": minor
+---
+
+시맨틱 stroke 컬러 토큰을 업데이트합니다.
+
+**이름이 변경되는 stroke 토큰**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)